cypress/support/util.js

```diff
@@ -54,4 +54,339 @@ export function getDraggableSelector(draggableId) {
     return `[${draggable.id}="${draggableId}"]`;
   }
   return `[${draggable.id}]`;
-}
\ No newline at end of file
+}
+
+export const testRecordResponse = [
```

Comment:
Could this be more useful to be in it's own file and used as a cypress fixture? https://docs.cypress.io/api/commands/fixture#Accessing-Fixture-Data

Reply:
Oh yeah, that's a much better place to put it